docs/readthedocs/source/doc/Orca/Overview/install.md

```diff
@@ -44,6 +44,8 @@ conda activate py37
 ---
 ## Install BigDL Orca
 
+In this section, you can use `pip install` to install BigDL Orca package, which is the most recommended way to install.
```

Comment:
This section demonstrates how to install BigDL Orca via pip, which is the most recommended way.

Reply:
Fixed.